satnogsclient/settings.py

```diff
@@ -16,11 +16,18 @@ def _cast_or_none(func, value):
 GROUND_STATION_LON = _cast_or_none(float, environ.get('SATNOGS_STATION_LON', None))
 GROUND_STATION_ELEV = _cast_or_none(float, environ.get('SATNOGS_STATION_ELEV', None))
 
+PWD = path.dirname(path.realpath(__file__))
+OUTPUT_PATH = environ.get('SATNOGS_OUTPUT_PATH', '/tmp/.satnogs')
+if not os.path.exists(OUTPUT_PATH ):
+    os.makedirs(OUTPUT_PATH )
+
+CA_CERT =path.join(PWD, 'sub.class1.server.ca.pem')
```

Comment:
I fail to make this work locally. If the problem is indeed SNI then providing StartSSL's cert won't solve the problem.